packages/swarm-components/src/LinkButton.jsx

```diff
@@ -0,0 +1,107 @@
+// @flow
+import * as React from 'react';
+import Icon from './Icon';
+import {
+	FILLS,
+	getButtonType,
+	getSwarmSize,
+	getIconPosition,
+} from './utils/buttonUtils';
+
+export type LinkButtonProps = {
+	/**
+	 * The bordered button
+	 */
+	bordered?: boolean,
+
+	/**
+	 * Classes for additional styles to be applied
+	 */
+	className?: string,
+
+	/**
+	 * Indicates whether the button is disabled or not
+	 */
+	disabled?: boolean,
+
+	/**
+	 * Supports icon classes found at https://meetup.github.io/swarm-icons/
+	 */
+	icon?: string,
+
+	/**
+	 * Inverted styles for dark backgrounds
+	 */
+	inverted?: boolean,
+
+	/**
+	 * The neutral style
+	 */
+	neutral?: boolean,
+
+	/**
+	 * The function invoked when interacting with Button
+	 */
+	onClick?: () => {},
```

Comment:
Should we remove `onClick` here and add `href` instead and make it a required prop?

If the intent of using `LinkButton` instead of the `Button` is to provide a route navigation (both SSR and rehydrated), then I am not sure if letting devs intercept the click is a good idea - in those cases they should uses a `Button`. I might be wrong here and don't account for all use-cases necessarily.

Reply:
I removed `onClick` from the `LinkButton` props. I'm not requiring href because HTML 5 doesn't require it. The anchor tag is treated as a placeholder for a link when there is no href. So, href is treated as every other attribute to an anchor tag and passed along in `other` props.

Reply:
I don't feel strong on encouraging vs. enforcing and this fits our general approach to design system.